featuretools/__main__.py

```diff
@@ -1,8 +1,10 @@
 import click
 import pandas as pd
+import pkg_resources
 
 import featuretools
 from featuretools.primitives.utils import get_featuretools_root
+from featuretools.utils.cli_utils import get_installed_packages, get_sys_info
```

Comment:
let’s also make them importable from `featuretools.utils` and access them from there. that way if we change the `cli_utils.py` file name it doesnt cause problems

Reply:
Fixed.